CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Sending a QUERY message may fail with `Assertion failed: !m_unbind_message_sent` ([#5149](https://github.com/realm/realm-core/pull/5149), since v11.8.0)
 * Subscription names correctly distinguish an empty string from a nullptr ([#5160](https://github.com/realm/realm-core/pull/5160), since v11.8.0)
 * Converting floats/doubles into Decimal128 would yield imprecise results ([#5184](https://github.com/realm/realm-core/pull/5184), since v6.1.0)
+* Trying to intern a string multiple times may lead to duplicates ([#5196](https://github.com/realm/realm-core/pull/5196), since v10.2.0)
```

Comment:
For release notes, we aim to express the changes in end-user terms. Hopefully the SDK's should just be able to copy/paste these notes into their release notes that goes out to users. How would an enduser experience this bug?

Reply:
good point, thanks for noticing it. i'll rewrite it.